dashboards/google-kubernetes-engine/gke-optimization-signals.json

```diff
@@ -1,247 +1,276 @@
                 }
               }
             ],
             "thresholds": [],
             "timeshiftDuration": "0s",
             "yAxis": {
-              "label": "",
               "scale": "LINEAR"
             }
           }
-        },
-        "width": 3,
-        "xPos": 6,
-        "yPos": 2
+        }
       },
       {
-        "height": 5,
+        "yPos": 16,
+        "xPos": 12,
+        "height": 20,
+        "width": 12,
         "widget": {
-          "title": "AVG Recommended MEM vs AVG MEM Request [Deployments]",
+          "title": "AVG MEM Request vs AVG MEM Allocatable",
           "xyChart": {
             "chartOptions": {
+              "displayHorizontal": false,
               "mode": "COLOR"
             },
             "dataSets": [
               {
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "timeSeriesQueryLanguage": "{ fetch\n    k8s_scale ::\n    kubernetes.io/autoscaler/container/memory/per_replica_recommended_request_bytes\n  | filter\n      namespace_name\n      !~\n      '(kube|istio|gatekeeper|gke|gmp|gke-gmp)-system|gke-managed-filestorecsi|gke-mcs'\n| ${project_id}\n| ${location}\n| ${cluster_name}\n  | group_by [cluster_name], .aggregate\n; fetch k8s_container::kubernetes.io/container/memory/request_bytes\n  | filter\n      namespace_name\n      !~\n      '(kube|istio|gatekeeper|gke|gmp|gke-gmp)-system|gke-managed-filestorecsi|gke-mcs'\n| ${project_id}\n| ${location}\n| ${cluster_name}\n  | group_by [cluster_name], .aggregate\n  }\n| ratio",
-                  "unitOverride": ""
+                  "prometheusQuery": "sum by (project_id, location, cluster_name) (\n  kubernetes_io:container_memory_request_bytes{\n    monitored_resource=\"k8s_container\",\n    ${project_id},\n    ${location},\n    ${cluster_name}\n  }\n)\n/ \n(\n  sum by (project_id, location, cluster_name) (\n    kubernetes_io:node_memory_allocatable_bytes{\n      monitored_resource=\"k8s_node\",\n      ${project_id},\n      ${location},\n      ${cluster_name}\n    }\n  ) and\n  sum by (project_id, location, cluster_name) (\n    kubernetes_io:node_memory_allocatable_bytes{\n      monitored_resource=\"k8s_node\",\n      ${project_id},\n      ${location},\n      ${cluster_name}\n    }\n  ) > 0\n)\n"
```

Comment:
Do you think the denominator from`sum by (ABC) / sum by(XYZ) and sum by(XYZ) > 0` be combined to just `sum by (ABC) / sum by(XYZ) > 0`? I can see the same results with this change for the bottom 4 widgets. Is that the right thing to do? That would make the query a bit cleaner.